scripts/piezomotor_config.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 18 Mar 2020
+
+@author: Philip Winkler
+
+Copyright © 2020, Philip Winkler, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+
+
+Purpose of this script: Before using the piezomotor stage, the individual axes of the
+controller stack need to be configured. This involves mainly the assignment of
+an axis number. The firmware also provides a script to get certain parameters,
+which need to be added to the yaml file.
+TODO: do we need to run this script for the parameters or are they always the same and can be
+    hardcoded?
+'''
+from __future__ import division
+from odemis.driver.piezomotor import PMD401
+from odemis.model import Axis
+
+if __name__ == '__main__':
+    print("Piezomotor Stage Configuration\n=====================================\n\n")
+
+    axis = Axis()
+    stage = PMD401("PMD", "stage", "ttyUSB*", {"x": axis}, {"x": 0})
+    print("Please connect the controller board for the x-axis and press enter. " +
+          "Don't connect any of the other boards yet.\n")
+    while True:
+        input()
+        try:
+            res = stage.set_axis_address(1)
+            break
+        except Exception as ex:
+            print("\nConfiguration failed, please try again\nException: %s" % ex)
```

Comment:
Don't you want to use some timeout in here?
Maybe have:
```
startup_time = time.time()
while not timeout(10, startup_time):
```
with:
```
def timeout(a):
  return [True if startup_time - time.time() < -a else False][0]
```

(But than not so ugly)